image_geometry/src/pinhole_camera_model.cpp

```diff
@@ -1,5 +1,5 @@
 #include "image_geometry/pinhole_camera_model.h"
-#include <sensor_msgs/distortion_models.h>
+#include <sensor_msgs/distortion_models.hpp>
```

Comment:
is that something we are planning on submitting upstream (the file renaming I mean)?